pallets/frequency-tx-payment/src/lib.rs

```diff
@@ -205,6 +219,33 @@ pub mod pallet {
 
 			call.dispatch(origin)
 		}
+
+		#[pallet::call_index(1)]
+		#[pallet::weight({
+		let dispatch_infos = calls.iter().map(|call| call.get_dispatch_info()).collect::<Vec<_>>();
+		let dispatch_weight = dispatch_infos.iter()
+				.map(|di| di.weight)
+				.fold(Weight::zero(), |total: Weight, weight: Weight| total.saturating_add(weight));
+
+		let capacity_overhead = T::DbWeight::get().reads(2)
+			.saturating_add(
+				T::DbWeight::get().writes(1)
+			);
+		let total = capacity_overhead.saturating_add(dispatch_weight);
+		(< T as Config >::WeightInfo::pay_with_capacity().saturating_add(total), DispatchClass::Normal)
```

Comment:
I just wanted to remind you that we added this quickly to have something. I recommend adding the benchmarks and running them.

Reply:
Thank you, great catch